### PR TITLE
docs(cheatsheet): add Directive to cheatsheet

### PR DESCRIPTION
**Pull request by cebor**
- **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit-message-format
- [x] Docs have been added / updated (for bug fixes / features)
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
  Doc Update: Adds Directive to the class decorators in cheatsheet

### CODE REVIEW

File: modules/angular2/docs/cheatsheet/class-decorators.md [hunk 1]
```diff
@@ -8,6 +8,18 @@ Class decorators
...
 syntax(ts):
+`@Directive({...})
+class MyDirective() {}`|`@Directive({...})`
+syntax(js):
+`var MyDirective = ng.core.Directive({...}).Class({...})`|`ng.core.Directive({...})`
+syntax(dart):
+`@Directive(...)
+class MyDirective() {}`|`@Directive(...)`
+description:
+Declares that a class is a directive and provides metadata about the component.
```
Comment (IgorMinar) [inline modules/angular2/docs/cheatsheet/class-decorators.md]: "about the directive."

File: modules/angular2/docs/cheatsheet/class-decorators.md [hunk 2]
```diff
@@ -8,6 +8,18 @@ Class decorators
 
 @cheatsheetItem
 syntax(ts):
+`@Directive({...})
```
Comment (IgorMinar) [inline modules/angular2/docs/cheatsheet/class-decorators.md]: Can you please move this under Components. Components are by far more common than Directives?

**Comment by IgorMinar** [comment]
can you please make the changes?

**Comment by cebor** [comment]
@IgorMinar done :)

**Comment by IgorMinar** [comment]
thanks!

**Pull request closed without merging**